src/content/en/updates/2018/07/chrome-69-media-updates.md

```diff
@@ -0,0 +1,311 @@
+`MediaKeySession` or fetch a real license. It does not require `MediaKeys` to be
+attached to any audio or video elements either.
+
+The HDCP Policy Check API works simply by calling
+`mediaKeys.getStatusForPolicy()` with an object that has a `minHdcpVersion` key
+and a valid value. If HDCP is available at the specified version, the returned
+promise resolves with a `MediaKeyStatus` of `'usable'`. Otherwise, the promise
+resolves with [other error values] of `MediaKeyStatus` such as
+`'output-restricted'` or `'output-downscaled'`. If the key system does not
+support HDCP Policy Check at all (e.g. Clear Key System), the promise rejects.
+
+In a nutshell, here’s how the API works for now. Check out the [official sample]
+to try out all versions of HDCP.
+
+    const config = [{
+      videoCapabilities: [{
+        contentType: 'video/webm; codecs="vp09.00.10.08"',
+        robustness: 'SW_SECURE_DECODE' // Widevine L3
+      }]
+    }];
+
+    navigator.requestMediaKeySystemAccess('com.widevine.alpha', config)
+    .then(mediaKeySystemAccess => mediaKeySystemAccess.createMediaKeys())
+    .then(mediaKeys => {
+
+      // Get status for HDCP 2.2
+      return mediaKeys.getStatusForPolicy({ minHdcpVersion: 'hdcp-2.2' })
+      .then(status => {
+        if (status !== 'usable')
+          return Promise.reject(status);
+
+        console.log('HDCP 2.2 can be enforced.');
+        // TODO: Fetch high resolution protected content...
+      });
+    })
+    .catch(error => {
+      // TODO: Fallback to fetch license or stream low-resolution content...
+    });
+
+### Available for Origin Trials
+
+In order to get feedback from web developers, the HDCP Policy Check API is
+available as an [Origin Trial] in Chrome 69 for Desktop (Chrome OS, Linux, Mac,
+and Windows). You will need to [request a token], so that the feature would be
+automatically enabled for your origin for a limited period of time, without the
+need to enable the experimental "Web Platform Features" flag at
+`chrome://flags/#enable-experimental-web-platform-features`.
+
+[Intent to Experiment](https://www.google.com/url?q=https://groups.google.com/a/chromium.org/forum/%23!topic/blink-dev/ITzZ_yx4bF8&sa=D&ust=1530528558634000&usg=AFQjCNFc5O4gFlE5rBSytmc73mGgqCyNgA) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5652917147140096) &#124;
+[Chromium Bug](https://crbug.com/709348)
+
+## MSE PTS/DTS compliance {: #pts }
+
+Buffered ranges and duration values are now reported by Presentation Time Stamp
+(PTS) intervals, rather than by Decode Time Stamp (DTS) intervals in [Media
+Source Extensions (MSE)].
+
+At the very beginnings of MSE, Chrome’s implementation was tested against WebM
+and MP3, some media stream formats where there was no such distinction between
+PTS and DTS. And it was working fine… until ISO BMFF (aka MP4) was added. This
+frequently contains out-of-order presentation versus decode time streams (eg,
```

Comment:
I don't know what you mean by 'out-of-order presentation versus decode time streams'.

The abbreviation e.g. should contain periods. Pro tip: e.g. and i.e. are easy to mix up. For the of clarity I always use 'for example' and 'in other words'.

Reply:
Done.